### DISCUSSION TITLE
Any limitations with long notes?

### DISCUSSION THREAD

**Discussion by jamie9090** [brianpetro/obsidian-smart-connections#398] [General]
Hey there. 

I have some notes that are long lists of Tasks or contain up to 10K words. 

Would this still be processed properly? 

Are there anythings to keep on mind where a note will not be fully processed?

Cheers!

**Comment by brianpetro** [comment]
@jamie9090 

If you simply want to include the full note as context, you'll want to specify the note using the `[[LINK]]` syntax in the chat.

If you're using a model that has a large enough context window, then the entire note should be included.

🌴
